spec/functional/resource/dsc_script_spec.rb

```diff
@@ -67,8 +67,7 @@ def delete_user(target_user)
     node = Chef::Node.new
     node.automatic['platform'] = 'windows'
     node.automatic['platform_version'] = '6.1'
-    node.automatic['kernel'][:machine] =
-      is_i386_process_on_x86_64_windows? ? :x86_64 : :i386
+    node.automatic['kernel'][:machine] = :x86_64  # Only 64-bit process is supported
```

Comment:
The failure case was under a 64-bit OS - we would specify a i386
architecture if NOT running as a 32-bit process under a 64-bit OS.  When we
ran under 32-bit, that was as a 64-bit process as you indicate.  So I think
process is correct.  Maybe we can say "62-bit architecture" instead?

On Thu, Aug 20, 2015 at 9:16 AM, Adam Edwards <notifications@github.com>
wrote:

> In spec/functional/resource/dsc_script_spec.rb
> <https://github.com/chef/chef/pull/3797#discussion_r37549239>:
>
> > @@ -67,8 +67,7 @@ def delete_user(target_user)
> >      node = Chef::Node.new
> >      node.automatic['platform'] = 'windows'
> >      node.automatic['platform_version'] = '6.1'
> > -    node.automatic['kernel'][:machine] =
> > -      is_i386_process_on_x86_64_windows? ? :x86_64 : :i386
> > +    node.automatic['kernel'][:machine] = :x86_64  # Only 64-bit process is supported
>
> So isn't the key here that we only support running these specs on a 64-bit
> OS vs. a 64-bit process? The specs themselves have always been 32-bit to
> this point, but Chef (also 32-bit) shells out to 64-bit).
>
> My note here is only about the comment, basically I think it's more
> accurate to say 64-bit OS instead of 64-bit process. Does that seem
> reasonable?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef/chef/pull/3797/files#r37549239>.
>